lib/phoenix/live_dashboard/router.ex

```diff
@@ -333,7 +333,8 @@ defmodule Phoenix.LiveDashboard.Router do
     ecto_session = %{
       repos: ecto_repos(ecto_repos),
       ecto_psql_extras_options: ecto_psql_extras_options,
-      ecto_mysql_extras_options: ecto_mysql_extras_options
+      ecto_mysql_extras_options: ecto_mysql_extras_options,
+      ecto_migrations_paths: ["../../bitfo/brain/priv/repo/migrations"]
```

Comment:
Need a way to pass this in, this was for my local dev

Reply:
It will be default if none are passed, https://github.com/phoenixframework/phoenix_live_dashboard/pull/371/commits/14aa9cfc00cc5b97e484b7da16367213d590fa07#diff-8bf276c87eadc1896bae41eaf34b8d0f654893dc9cd7792eb8650c5079222417R123

This is primarily an escape hatch if your migrations are located elsewhere, or in my case, testing the dashboard

Reply:
I was testing using my app's database like so `PG_DATABASE=brain_dev mix dev --postgres`, this will yield `** file not found **` without explicitly passing the path